.helm/templates/crd-cdm.yaml

```diff
@@ -0,0 +1,147 @@
+{{- if .Values.customResourceDefinitions.create }}
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: cdms.streaming.sneaksanddata.com
+spec:
+  group: streaming.sneaksanddata.com
+  scope: Namespaced
+  names:
+    plural: cdms
+    singular: cdm
+    kind: CdmChangeFeed
+    shortNames:
+      - cdms
+  versions:
+    - name: v1beta1
+      served: true
+      storage: true
+      additionalPrinterColumns:
+        - name: Source location
+          type: string
+          jsonPath: .spec.baseLocation
+        - name: Entity
+          type: string
+          jsonPath: .spec.entityName
+        - name: Refresh Interval
+          type: string
+          jsonPath: .spec.changeCaptureIntervalSeconds
+        - name: Sink location
+          type: string
+          jsonPath: .spec.sinkLocation
+        - name: Phase
+          type: string
+          jsonPath: .status.phase
+      subresources:
+        status: {}
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            spec:
+              type: object
+              properties:
+                connectionStringRef:
+                  description: |
+                    Name of the secret containing the connection string.
+                    Should have a key named 'ARCANE_CONNECTIONSTRING'.
+                  type: object
+                  properties:
+                    name:
+                      type: string
+                jobTemplateRef:
+                  description: |
+                    Name of the job template to be used for the streaming job if stream is running in normal mode.
+                  type: object
+                  properties:
+                    name:
+                      type: string
+                    kind:
+                      type: string
+                    apiGroup:
+                      type: string
+                  default:
+                    apiGroup: streaming.sneaksanddata.com
+                    kind: StreamingJobTemplate
+                    name: standard-job
+                reloadingJobTemplateRef:
+                  description: |
+                    Name of the job template to be used for the streaming job if stream is running in reloading mode.
+                  type: object
+                  properties:
+                    name:
+                      type: string
+                    kind:
+                      type: string
+                    apiGroup:
+                      type: string
+                  default:
```

Comment:
Remove the defaults here.

Reply:
Or make sure that default value matches the name of the template installed by the Helm chart. This would be better.